pl_bolts/models/detection/yolo/yolo_config.py

```diff
@@ -0,0 +1,265 @@
+import re
+from typing import List, Tuple
+from warnings import warn
+
+import torch.nn as nn
+from pytorch_lightning.utilities.exceptions import MisconfigurationException
+
+from pl_bolts.models.detection.yolo import yolo_layers
+
+
+class YoloConfiguration:
+    """
+    This class can be used to parse the configuration files of the Darknet YOLOv4 implementation.
+    The `get_network()` method returns a PyTorch module list that can be used to construct a YOLO
```

Comment:
Single quote will be rendered in italic in the docs i.e. *get_network()*, so we need double quotes here to make it look like `get_network()`.
```suggestion
    The ``get_network()`` method returns a PyTorch module list that can be used to construct a YOLO
```

Reply:
I replaced single quotes with double quotes in several docstrings.